### DISCUSSION TITLE
"Links" in sql file

### DISCUSSION THREAD

**Discussion by artDeil4ek** [trickerer/Trinity-Bots#464] [Q&A] [answered]
Hello. I searched an answer for my question, but didn't find it. In the file "2023_04_16_00_creature_template_npcbot_wander_nodes.sql" we have lines like this: 
INSERT INTO 'creature_template_npcbot_wander_nodes'('id','name','mapid','zoneid','areaid','minlevel','maxlevel','flags','x','y','z','o',**links**) VALUES

In general, everything is clear here. XYZ coordinates, zoneid, etc. What does **_links_** mean? What are they referring to?
(2372,'WarsongGulchSpawnA1',489,3277,4571,10,80,251,1535.77,1481.37,352.109,0.188495,'**2373:0 2375:0 2378:0 2377:0** ')
Thank you.

**Comment by Russianranger** [comment, marked as answer]
"Links" from a waypoint are the other waypoints connected to it. For example - I make 3 waypoints in a triangle. They are 001, 002 and 003 respectively. To have a bot travel from 001 to 002, I need to "link" them. So when you look at 001, it should show "Links 002:0". And consequently looking at 002 should show "links 001:0". I can then attach 003 to both 001 and 002. So the bot can effectively travel between the 3 waypoints.
